benchmark/benchmark.c

```diff
@@ -164,143 +196,150 @@ static inline void seL4_BenchmarkTrackDumpSummary(benchmark_track_kernel_entry_t
 
 void notified(microkit_channel ch)
 {
-    switch(ch) {
-        case START:
-            sel4bench_reset_counters();
-            THREAD_MEMORY_RELEASE();
-            sel4bench_start_counters(benchmark_bf);
+    switch (ch) {
+    case START:
+        sel4bench_reset_counters();
+        THREAD_MEMORY_RELEASE();
+        sel4bench_start_counters(benchmark_bf);
 
-            #ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
-            microkit_benchmark_start();
-            #endif
+#ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
+        microkit_benchmark_start();
+#endif
 
-            #ifdef CONFIG_BENCHMARK_TRACK_KERNEL_ENTRIES
-            seL4_BenchmarkResetLog();
-            #endif
```

Comment:
Likewise here.

Reply:
Can’t do it for this one since its part of libsel4